custom/icds_reports/utils/aggregation_helpers/distributed/availing_service_forms.py

```diff
@@ -0,0 +1,85 @@
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_AVAILING_SERVICES_TABLE
+from custom.icds_reports.utils.aggregation_helpers import month_formatter
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import (
+    StateBasedAggregationDistributedHelper,
+)
+
+
+class AvailingServiceFormsAggregationDistributedHelper(StateBasedAggregationDistributedHelper):
+    helper_key = 'availing_service-forms'
+    ucr_data_source_id = 'static-availing_service_form'
+    aggregate_parent_table = AGG_AVAILING_SERVICES_TABLE
+
+    def data_from_ucr_query(self):
+        month = self.month.replace(day=1)
+        current_month_start = month_formatter(self.month)
+        next_month_start = month_formatter(self.month + relativedelta(months=1))
+
+        query_params = {
+            "month": month_formatter(month),
+            "state_id": self.state_id,
+            "current_month_start": current_month_start,
+            "next_month_start": next_month_start,
+        }
+        return """
+        SELECT DISTINCT
+            %(state_id)s AS state_id,
+            supervisor_id,
+            awc_id,
+            %(month)s::DATE AS month,
+            person_case_id AS person_case_id,
+            LAST_VALUE(is_registered) OVER w AS is_registered,
+            timeend AS registration_date
```

Comment:
this needs to also be `LAST_VALUE` or else it is creating one row per form rather than one row per case